cardano-cli/CHANGELOG.md

```diff
@@ -1,5 +1,72 @@
 # Changelog for cardano-cli
 
+## 8.23.0.0
+
+- Introduces the `governance action create-hardfork` cmd.
+  (feature, test)
+  [PR 746](https://github.com/IntersectMBO/cardano-cli/pull/746)
+
+- Export friendly function that returns ByteString + export buildShelleyAddress
+  (compatible)
+  [PR 748](https://github.com/IntersectMBO/cardano-cli/pull/748)
+
+- Export toTxOutInAnyEra
+  (compatible)
+  [PR 745](https://github.com/IntersectMBO/cardano-cli/pull/745)
+
+- Enable manual specification of execution units for plutus voting and proposing scripts
+  (feature, breaking)
+  [PR 744](https://github.com/IntersectMBO/cardano-cli/pull/744)
+
+- query stake-distribution: add --output-{json,text} flags
+  (feature, breaking)
+  [PR 743](https://github.com/IntersectMBO/cardano-cli/pull/743)
+
+- Introduce new `cardano-cli latest transaction build-estimate` command which will
+  produce a balanced transaction body without requiring a connection to a live node
+  (feature)
+  [PR 728](https://github.com/IntersectMBO/cardano-cli/pull/728)
+
+- Add `minFeeRefScriptCostPerByte`  to create-protocol-parameter-update governance action command.
+  (feature)
+  [PR 736](https://github.com/IntersectMBO/cardano-cli/pull/736)
+
+- Fixed misunderstanding in generated README file.
+  (bugfix)
+  [PR 726](https://github.com/IntersectMBO/cardano-cli/pull/726)
+
+- Update cardano-api to 8.45.0.0
+  (compatible)
+  [PR 720](https://github.com/IntersectMBO/cardano-cli/pull/720)
+
+- conway governance committe key-hash: support extended CC keys
+  (feature, compatible, bugfix)
+  [PR 717](https://github.com/IntersectMBO/cardano-cli/pull/717)
+
+- Shorten and modernize some governance tests
+  (improvement, test)
+  [PR 714](https://github.com/IntersectMBO/cardano-cli/pull/714)
+
+- Provide a default value for `calculate-min-fee --reference-script-size`
+  (improvement)
+  [PR 716](https://github.com/IntersectMBO/cardano-cli/pull/716)
+
+- key non-extended-key: make pattern matches exhaustive, so that we are notified of possible extensions
+  (improvement)
+  [PR 710](https://github.com/IntersectMBO/cardano-cli/pull/710)
+
+- Use mapSomeAddressVerificationKey (provided by API) instead of foldSomeAddressVerificationKey
+  (improvement)
+  [PR 708](https://github.com/IntersectMBO/cardano-cli/pull/708)
+
+- Enable both `--cold-script-file` and `--cold-script-hash` for `governance committee create-hot-key-authorization-certificate` and `committee create-cold-key-resignation-certificate`
+  (feature, compatible)
+  [PR 699](https://github.com/IntersectMBO/cardano-cli/pull/699)
+
+- Fix inversion in --include-stake parser + print "stake" field even when there is no stake (do not hide stake absence)
```

Comment:
This could use some more elaboration as based on this entry even I'm not sure what has been fixed.

Reply:
Changed to `Fixed --include-stake behavior in query drep-state` :+1: